cdap-ui/app/hydrator/hydrator-modal.less

```diff
@@ -362,6 +362,26 @@
   }
 }
 
+.modal.hydrator-modal {
+  &.node-config-modal {
+    .modal-content {
+      .modal-body {
+        .console-type {
+          .bottompanel-body {
+            .form-horizontal {
+              .form-group {
+                .control-label {
+                  padding-right: 0;
```

Comment:
I think this can be moved to the `.control-label` class where you added the `font-weight: 500` property. I tested making the change locally and it works in both studio and detail.